android/src/main/java/com/getbouncer/RNCardVerifyModule.java

```diff
@@ -42,7 +48,116 @@ public RNCardVerifyModule(ReactApplicationContext reactContext) {
 
             @Override
             public void onActivityResult(Activity activity, int requestCode, int resultCode, Intent data) {
-                if (requestCode == SCAN_REQUEST_CODE) {
+                if (requestCode == SCAN_REQUEST_CODE && useLocalVerificationOnly) {
+                    com.getbouncer.cardverify.ui.local.CardVerifyActivity.parseVerifyResult(resultCode, data, new com.getbouncer.cardverify.ui.local.CardVerifyActivityResultHandler() {
+                        @Override
+                        public void cardScanned(
+                            @NotNull String cardPan,
+                            @Nullable String cardName,
+                            @Nullable String cardExpiryMonth,
+                            @Nullable String cardExpiryYear,
+                            boolean isCardValid,
+                            @Nullable String cardValidationFailureReason,
+                            @Nullable String cardValidationDetails
+                        ) {
+                            final String issuer = PaymentCardUtils.getCardIssuer(cardPan).getDisplayName();
+                            final WritableMap cardMap = new WritableNativeMap();
+                            cardMap.putString("number", cardPan);
+                            cardMap.putString("expiryDay", null);
+                            cardMap.putString("expiryMonth", cardExpiryMonth);
+                            cardMap.putString("expiryYear", cardExpiryYear);
+                            cardMap.putString("issuer", issuer);
+                            cardMap.putString("cvc", null);
+                            cardMap.putString("cardholderName", cardName);
+                            cardMap.putString("payloadVersion", "2");
+                            cardMap.putString("verificationPayload", null);
+                            cardMap.putBoolean("isCardValid", isCardValid);
+                            cardMap.putString("cardValidationFailureReason", cardValidationFailureReason);
+                            cardMap.putString("cardValidationDetails", cardValidationDetails);
```

Comment:
Can you tell me the types of these values so I can match them?

Reply:
yep! that's the plan. here are the possible values:

[cardValidationFailureReason](https://github.com/getbouncer/cardverify-android/blob/master/cardverify-ui/src/main/java/com/getbouncer/cardverify/ui/local/result/CompletionLoopAggregator.kt#L18) is one of the following:
- `ModelDownloadFailure`
- `ValidationFailure`
- `ProcessingFailure`